### PR TITLE
coil support

### PR DESCRIPTION
**Pull request by anselm**
This is support for coil. It adds coil to the list of plugins and then also I've pasted the latest rev of coil itself into the web plugins folder.

### CODE REVIEW

**Comment by bluemarvin** [comment]
What are the performance implications for adding this web extension to every page load?

File: app/src/main/assets/web_extensions/coil/static/goSettings.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+const url = 'https:/coil.com/settings'
```
Comment (jonathansampson) [inline app/src/main/assets/web_extensions/coil/static/goSettings.js]: Shouldn't that be `http://` (note, two forward slashes)?

Comment (anselm) [reply app/src/main/assets/web_extensions/coil/static/goSettings.js]: wow good catch - i will forward to that team

Comment (anselm) [reply app/src/main/assets/web_extensions/coil/static/goSettings.js]: https://github.com/coilhq/web-monetization-projects/blob/master/packages/coil-extension/static/goSettings.js was the original location - am advising them

**Pull request closed without merging**